### ISSUE TITLE
Fury of Illidari / Artifact Abilities not working

### ISSUE THREAD

**Issue by Fablezim** [AdiAddons/AdiButtonAuras#240]
First let me say I just discovered this and it is AMAZING, great job, I love it :)

#### Which version of AdiButtonAuras are you using (enter the exact version number here)?
Updated today, 2.0.3 Release

#### How did you install the addon (curse client/manually)?
Curse

#### If using the curse client, what type of release did you install (alpha/beta/release)?
Release

#### If you installed manually, where did you download the addon from?
N/A

#### Which type of installation did you choose (nolib/normal)?
Normal

#### If your report is about a missing or wrong spell provide the following information:
  - Your class: Demon Hunter
  - Your spec: Havoc
  - Name of the spell (and spell id if known): Fury of the Illidari, ID#: 201628
  - Source of the spell (core ability/talent/artifact/pvp): Artifact

#### If you have an error report, copy it below:
Date: 2016-10-05 10:53:54
ID: 1
Error occured in: Global
Count: 1
Message: ..\AddOns\AdiButtonAuras\core\RuleDSL.lua line 195:
   Invalid spell identifier: 114866
Debug:
   (tail call): ?
   [C]: ?
   AdiButtonAuras\core\RuleDSL.lua:95:
      AdiButtonAuras\core\RuleDSL.lua:90
   AdiButtonAuras\core\RuleDSL.lua:123:
      AdiButtonAuras\core\RuleDSL.lua:122
   AdiButtonAuras\core\RuleDSL.lua:195:
      AdiButtonAuras\core\RuleDSL.lua:194
   [C]: ?
   AdiButtonAuras\core\Core.lua:388: ?()
   ...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:145:
      ...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:145
   [string "safecall Dispatcher[1]"]:4:
      [string "safecall Dispatcher[1]"]:4
   [C]: ?
   [string "safecall Dispatcher[1]"]:13: ?()
   ...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:90: Fire()
   ...ttonAuras\libs\LibSpellbook-1.0\LibSpellbook-1.0.lua:346:
      ...ttonAuras\libs\LibSpellbook-1.0\LibSpellbook-1.0.lua:308
   (tail call): ?
Locals:
None

This error doesn't seem to be or cause any real problems, though it does come appear when I log in or reload UI. The problem is the addon has no information about Fury of the Illidari ability. This also happens with my mage and the same issue with her artifact ability, Phoenix Flames (ID#: 194466), thats the only other character I've tested with. I do have LibArtifactData and LibPlayerData installed and updated, even tried the alpha release of them but to no avail. Am I doing something wrong? Thanks in advance!

**Comment by Rainrider** [comment]
First about the error report:
114866 is the spell id of Soul Reaper, which was removed in Legion. Please check your custom rules and disable/delete/update any of them that contain it. You maybe accidentally enabled our sample rule, which is outdated and that is causing the error.

About Fury of the Illidari:
LibPlayerSpells (LPS) actually has the ID, however there aren't any associated auras to it. Since ABA is an aura duration display, I'm not sure what you expect it to show/know about Fury of the Illidari.

Same for Phoenix's Flames. If it does not apply a (de)buff, then ABA does not care about it.

**Comment by Fablezim** [comment]
Ah you're right, I had a faulty script active I forgot to delete. And thanks, I didn't realize it was AURA based, which I should have gathered from the title of the addon haha. 

However, for some reason now im getting this error:
Date: 2016-10-05 13:02:06
ID: 1
Error occured in: Global
Count: 1
Message: ...e\AddOns\Blizzard_ArtifactUI\Blizzard_ArtifactUI.lua line 41:
   attempt to compare number with nil
Debug:
   [C]: ?
   ...e\AddOns\Blizzard_ArtifactUI\Blizzard_ArtifactUI.lua:41: ArtifactUI_CanViewArtifact()
   ...ddOns\Blizzard_ArtifactUI\Blizzard_ArtifactPerks.lua:680: EvaluateRelics()
   ...ddOns\Blizzard_ArtifactUI\Blizzard_ArtifactPerks.lua:541:
      ...ddOns\Blizzard_ArtifactUI\Blizzard_ArtifactPerks.lua:539
   [C]: SetShown()
   ...e\AddOns\Blizzard_ArtifactUI\Blizzard_ArtifactUI.lua:172: SetTab()
   ...e\AddOns\Blizzard_ArtifactUI\Blizzard_ArtifactUI.lua:138: EvaulateForgeState()
   ...e\AddOns\Blizzard_ArtifactUI\Blizzard_ArtifactUI.lua:70:
      ...e\AddOns\Blizzard_ArtifactUI\Blizzard_ArtifactUI.lua:67
   [C]: Show()
   ..\FrameXML\UIParent.lua:2336: SetUIPanel()
   ..\FrameXML\UIParent.lua:2154: ShowUIPanel()
   ..\FrameXML\UIParent.lua:2048:
      ..\FrameXML\UIParent.lua:2044
   [C]: SetAttribute()
   ..\FrameXML\UIParent.lua:2830:
      ..\FrameXML\UIParent.lua:2818
   [C]: ShowUIPanel()
   ..\FrameXML\UIParent.lua:1466:
      ..\FrameXML\UIParent.lua:891

The artifact tree window also auto-opens upon login and reload. I used to have the same problem with Exorsus Raid Tools but disabled it, now its happening here. (I disabled AdiButtonAuras and the problem went away.) Should I post in a new thread?
